dialoptions.go

```diff
@@ -658,19 +658,23 @@ func WithResolvers(rs ...resolver.Builder) DialOption {
 	})
 }
 
-// WithSharedRecvBufferPool returns a DialOption that configures the ClientConn
+// WithRecvBufferPool returns a DialOption that configures the ClientConn
 // to use the provided shared buffer pool for parsing incoming messages. Depending
 // on the application's workload, this could result in reduced memory allocation.
 //
 // If you are unsure about how to implement a memory pool but want to utilize one,
-// begin with grpc.NewSimpleSharedBufferPool.
+// begin with grpc.NewsimpleSharedBufferPool.
```

Comment:
Sorry if I wasn't clear; I wanted this renamed to: `grpc.NewSharedBufferPool`

Reply:
Done with 96f5a276daf93ec80162d7e58903abef081a1c72.